community/pike/APKBUILD

```diff
@@ -0,0 +1,187 @@
+# Maintainer: Paweł Tomak <pawel@tomak.eu>
+pkgname=pike
+pkgver=8.0.610
+pkgrel=0
+pkgdesc="Pike Programing language"
+url="https://pike.lysator.liu.se"
+arch="x86_64"
+license="GPL-2.0-or-later"
+options="!check" # there is no tests for those packages
+depends=
+makedepends="gmp-dev
+	bzip2-dev
+	freeglut-dev
+	gdbm-dev
+	giflib-dev
+	giflib-utils
+	gtk+2.0
+	libglade-dev
+	libsm-dev
+	mesa-gl
+	nettle
+	nettle-dev
+	nettle-utils
+	pcre-dev
+	sdl-dev
+	sdl_mixer-dev
+	sqlite-dev
+	sqlite-libs
+	readline
+	zlib-dev
+	librsvg-dev"
+install=
+subpackages="$pkgname-doc $pkgname-full"
+source="http://pike.lysator.liu.se/pub/pike/all/${pkgver}/Pike-v${pkgver}.tar.gz"
+builddir="$srcdir/Pike-v$pkgver"
```

Comment:
```
source="http://pike.lysator.liu.se/pub/${pkgname}/all/${pkgver}/${_pkgname}-v${pkgver}.tar.gz"
builddir="${srcdir}/${_pkgname}-v${pkgver}"
```

Reply:
Please don't promote unnecessary curly braces around variables, respect our code-style.

Reply:
I still don't get why you insist on everyone keeping these variable parsing rules in our heads, but I'll edit it to respect the style as you requested.